packages/docs/src/pages/en/getting-started/contributing.md

```diff
@@ -54,120 +66,59 @@ yarn
 yarn build
 ```
 
-The build process compiles all of the Vuetify packages for development and may take awhile (grab some ☕). Once the packages are built, start your local development server by running <kbd>yarn dev</kbd> in the terminal.
+The build process compiles all the Vuetify packages for development and may take a while (grab some ☕). Once the packages are built, you can start developing.
 
 ## Starting your environment
 
-There are a few ways to start the development environments in the Vuetify mono-repo.
-
-```bash
-# All commands run from root directory
-
-# To start vuetify dev
-yarn dev
+There are a few ways to start the development environments in the Vuetify monorepo.
 
-# To access your local vuetify playground: http://localhost:8080
+### Vuetify
 
-# To start docs
-yarn dev docs
+The Vuetify library is located in `packages/vuetify`. In `packages/vuetify/dev` you will find a `Playground.vue` file; running `yarn dev` from the project root will start a dev server on [localhost:8080](http://localhost:8080/) with this file loaded. You can test your changes in the playground then copy its contents into your pull request when you're ready.
 
-# To access your local docs environment: http://localhost:8095
+You can also test vuetify in your own project using [`yarn link`](https://classic.yarnpkg.com/en/docs/cli/link/):
 
-# To start a specific package
-yarn dev <package name>
+- Navigate to `packages/vuetify`
+- Run `yarn link`
+- Navigate to your project's directory
+- Run `yarn link vuetify`
 
-# To build all packages
-yarn build
+If your project is using vuetify-loader you will have to run `yarn build:lib` in the vuetify package to see changes, otherwise you can use `yarn watch` for incremental builds.
 
-# To build a specific package
-yarn build <package name>
+### Documentation
 
-# Package alias
-api-generator: api
-```
+The documentation is located in `packages/docs` but also uses some files from `packages/api-generator`. A dev server for the documentation can be started by running `yarn dev:docs` from the project root and will be available on [localhost:8095](http://localhost:8095/) by default.
```

Comment:
```suggestion
The documentation is located in `packages/docs` but also uses some files from `packages/api-generator`. A dev server for the documentation can be started by running `yarn dev docs` from the project root and will be available on [localhost:8095](http://localhost:8095/) by default.
```

Reply:
or we could also just update the dev/build scripts, either or

Reply:
I always run those from the packages anyway so I forgot the structure. You can push to this branch if you like I just PRed it because I'm going to bed and didn't want the work to be duplicated.

Reply:
That's fine, I don't have access to the repo at the moment, just leaving notes. Ill poke it after work. Thanks for starting this at least. Sleep well!